ui/seek_bar.js

```diff
@@ -583,6 +580,201 @@ shaka.ui.SeekBar = class extends shaka.ui.RangeElement {
     this.thumbnailTime_.textContent = '';
   }
 
+  /**
+   * Sets up the chapter element creator and change handling.
+   * @private
+   */
+  setupChapters_() {
+    /** @type {!AbortController} */
+    let abortController = new AbortController();
+    let language = 'und';
+    /** @type {!Array<shaka.extern.Chapter>} */
+    let chapters = [];
+
+    /**
+    * Does a value compare on chapters.
+    * @param {shaka.extern.Chapter} a
+    * @param {shaka.extern.Chapter} b
+    * @return {!boolean}
+    */
+    const chaptersEqual = (a, b) => {
+      return !(a && b) ||
+     (a.id === b.id && a.title === b.title &&
+     a.startTime === b.startTime && a.endTime === b.endTime);
+    };
+
+    /** @type {function(): void} */
+    const handleChapterTrackChange = () => {
+      const nextLanguage = this.player.getCurrentTextLanguage() || 'und';
```

Comment:
You have to use the language of the UI not of a text track that may or may not exist.

Reply:
Where do I get the language of the UI? I don't see a config value or property containing it.